src/components/Rivet.tsx

```diff
@@ -127,49 +131,86 @@ const parseRivetSpacing = (type, style?: Size | BoxStyle) =>
 
-const parseRivetPadding = (props: Props) =>
-    parseRivetSpacing('p', props.rvtPadding);
+const parseRivetMargin = (margin?: Size | BoxStyle) =>
+    parseRivetSpacing('m', margin);
 
-const parseRivetTypescale = (props: Props) =>
-    props.rvtTypescale 
-    ? `rvt-ts-${props.rvtTypescale}`
+const parseRivetPaddingLegacy = ({rvtPadding}: Props) =>
+    parseRivetSpacing('p', rvtPadding);
+
+const parseRivetPadding = (padding?: Size | BoxStyle) =>
+    parseRivetSpacing('p', padding);
+
+const parseRivetTypescaleLegacy = ({ rvtTypescale }: Props) =>
+    rvtTypescale 
+    ? `rvt-ts-${rvtTypescale}`
     : "";
 
-const parseRivetBorder = (props: Props) => 
-    props.rvtBorder
-    ? Array.isArray(props.rvtBorder)
-        ? props.rvtBorder.map((value) => `rvt-border-${value}`).join(" ")
-        : `rvt-border-${props.rvtBorder}`
+const parseRivetTypescale = (typescale?: Typescale) => ({
+    [`rvt-ts-${typescale}`]: !!typescale
+});
+
+const parseRivetBorderLegacy = ({ rvtBorder }: Props) => 
+    rvtBorder
+    ? Array.isArray(rvtBorder)
+        ? rvtBorder.map((value) => `rvt-border-${value}`).join(" ")
+        : `rvt-border-${rvtBorder}`
     :"";
 
-const parseRivetDisplay = (props: Props) => {
-    switch(props.rvtDisplay) {
+const parseRivetBorder = (border?: Border) => {
+    if(Array.isArray(border)) {
+        return border.map((value) => `rvt-border-${value}`);
+    } else {
+        return ({
+            [`rvt-border-${border}`]: !!border
+        });
+    }
+}
+
+const parseRivetDisplayLegacy = ({ rvtDisplay }: Props) => {
+    switch(rvtDisplay) {
         case "inline":
         case "inline-block":
         case "block":
         case "flex":
-            return `rvt-display-${props.rvtDisplay}`;
+            return `rvt-display-${rvtDisplay}`;
         case "flex-vertical-center":
             return "rvt-display-flex rvt-vertical-center";
         default: return "";
     }
 }
 
-const parseRivetHidden= (props: Props) =>
-    props.rvtHide
-    ? `rvt-hide-${props.rvtHide}`
+const parseRivetDisplay = (display?: Display) => {
+    switch(display) {
+        case 'inline':
+        case 'inline-block':
+        case 'block':
+        case 'flex':
+            return `rvt-display-${display}`;
+        case 'flex-vertical-center':
+            return 'rvt-display-flex rvt-vertical-center';
+        default: return '';
+    }
+}
+
+const parseRivetHiddenLegacy = ({ rvtHide }: Props) =>
+    rvtHide
+    ? `rvt-hide-${rvtHide}`
     : "";
 
-const standardDecorators = [
-    parseRivetMargin,
-    parseRivetPadding,
-    parseRivetTypescale,
-    parseRivetBorder,
-    parseRivetDisplay,
-    parseRivetHidden
+const parseRivetHidden = (hide?: Hidden) => ({
+    [`rvt-hide-${hide}`]: !!hide
+});
+
+const legacyStandardDecorators = [
+    parseRivetMarginLegacy,
+    parseRivetPaddingLegacy,
+    parseRivetTypescaleLegacy,
+    parseRivetBorderLegacy,
+    parseRivetDisplayLegacy,
+    parseRivetHiddenLegacy
 ];
 
```

Comment:
Is the plan to get rid of these *Legacy decorators when everything's migrated from `classify` to `rivetize`?

Reply:
Yes, that is what I was thinking.